### PR TITLE
Styling updates for FAQ, as well as template adjustments for header/footer with updated styling

### LINKED ISSUE #58: Styling updates for FAQ to create better separation between questions

### PR DESCRIPTION
**Pull request by nickbrandt**
Resolves #58 

This is to add some additional spacing for the FAQ section to help with our anchored links (global typography updates).

Also included are adjustments to the header/footer templates and some styling to clean that up.

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            </li> -->
+            {%- if site.email -%}
+            <li>
+              <svg class="svg-icon">
+                <use xlink:href="{{relativeBase}}architecture/images/mail.svg#mail"></use>
+              </svg>
+              <a class="u-email" href="mailto:{{ site.email }}">{{ site.email }}</a></li>
+            {%- endif -%}
+        </ul>
+        {%- include social.html -%}
```
Comment (devinreams) [inline _includes/footer.html]: >jekyll 3.6.2 | Error:  Could not locate the included file 'social.html'

@nickbrandt are you able to build this locally? I realize this is probably directly from the underlying template...

Comment (nickbrandt) [reply _includes/footer.html]: @devinreams Hmmm...yea I have no issues building this locally.

**Comment by devinreams** [comment]
@nickbrandt here's what the `footer.html` included with minima 2.1.1 looks like for me. It doesn't "include" a social.html becuase it seems to have that stuff baked right in:

https://gist.github.com/devinreams/075b265928bbc1b8d5f7c81959a4f444

So I wonder if we have a version mismatch of the theme. What comes back out after you `bundle install` and then type `bundle info minima`?

**Comment by nickbrandt** [comment]
@devinreams Ah, looks like I'm using 2.4.1...where is it I can find these template files? I had initially just pulled from here...https://github.com/jekyll/minima
screen shot 2018-05-24 at 1 03 41 pm

**Comment by nickbrandt** [comment]
@devinreams apologies for the confusion...these templates should be good now. There is probably some refactoring I could do here in terms of overwriting some of their sass files vs. how I'm currently overriding them with my own styles.

**Comment by devinreams** [comment]
@nickbrandt Huh I wonder how you got a newer version of minima. I think the GitHub gem is what dictates what version will be available on GitHub Pages.

That aside, I pulled down the latest and it builds/runs fine here. But the footer still looks a bit disjointed to me. Is this what it looks like to you?

screen shot 2018-05-24 at 3 14 39 pm

**Pull request merged**